BootloaderCommonPkg/Include/Library/CryptoLib.h

```diff
@@ -49,12 +61,33 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 typedef UINT8 HASH_CTX[IPP_HASH_CTX_SIZE];   //IPP Hash context buffer
 
 //
-// RSA verify required key parameter format
+// PUB Key format for reference
 //
+//typedef struct {
+ // UINT8                    Modulus[0];
+// UINT8                    PubExp[4];
+//} PUB_KEY;

```

Comment:
Can we remove this comment block ?
It is more confusing to me than help.

Reply:
We would need to add reference some where about format we follow for Public Key (Modulus and Exp).  I would update as part of PUB_KEY_HDR comments.

Reply:
Sounds good to me.